build.sbt

```diff
@@ -46,7 +46,7 @@ lazy val scoverageSettings: Seq[Setting[_]] = Seq(
     ".*(BuildInfo|Routes|Options).*",
     "logger.*\\(.*\\)"
   ).mkString(";"),
-  coverageMinimum := 96,
+  coverageMinimum := 93,
```

Comment:
Please do not drop the coverage percentage.
It represents lines of code that have been covered.
It does not mean that we are testing all the possible outcomes.

Normally the Twirl views have some code that is more difficult to test, so we could pair on those if you need.